openstudiocore/src/model/EnergyManagementSystemActuator.hpp

```diff
@@ -54,6 +57,16 @@ class MODEL_API EnergyManagementSystemActuator : public ModelObject {
   //@{
 
   explicit EnergyManagementSystemActuator(const ModelObject& modelObject, const std::string actuatedComponentType, const std::string actuatedComponentControlType);
+  //These constructors below are for SpaceloadInstances that are defined in SpaceTypes that are used in Spaces.
```

Comment:
I think to get doxygen to pick up these comments you need to use the multiline comment style like this....
```
/** These constructors below are for SpaceloadInstances that are defined in SpaceTypes that are used in Spaces.
    * Upon translation, the SpaceLoadInstances use ZoneLists which are not avail in OS
    * The  ZoneListName is the SpaceType name
    * The Zone's are the Space->ThermalZone names
    * So to attach to a future zone, use the TZ or the Space that the SpaceLoadInstance will operate on
**/
```